applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -138,118 +77,52 @@ struct encoded_audio {
- *
- * @return	0 for success, error otherwise.
- */
-int le_audio_user_defined_button_press(enum le_audio_user_defined_action action);
-
-/**
- * @brief	Get configuration for audio stream.
- *
- * @param[out]	bitrate		Pointer to the bitrate used; can be NULL.
- * @param[out]	sampling_rate	Pointer to the sampling rate used; can be NULL.
- * @param[out]	pres_delay	Pointer to the presentation delay used; can be NULL.
- *
- * @retval	0		Operation successful.
- * @retval	-ENXIO		The feature is disabled.
- * @retval	-ENOTSUP	The feature is not supported,
- * @retval	error		Otherwise
- */
-int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate, uint32_t *pres_delay);
-
-/**
- * @brief	Set pointer to the connection.
- *
- * @note	Used by CIS.
- *
- * @param[in]	conn	The connection pointer.
- */
-void le_audio_conn_set(struct bt_conn *conn);
-
-/**
- * @brief	Set periodic advertising sync.
+ * @brief	Stop the LE Audio role given by @p role.
  *
- * @param[in]	pa_sync		Pointer to the periodic advertising sync.
- * @param[in]	broadcast_id	Broadcast ID of the periodic advertising.
+ * @param[in]	Type of role to stop, can only specify one at a time.
  *
  * @return	0 for success, error otherwise.
  */
-int le_audio_pa_sync_set(struct bt_le_per_adv_sync *pa_sync, uint32_t broadcast_id);
+int le_audio_stop(enum audio_roles role);
 
 /**
- * @brief	Notify about disconnected connection.
+ * @brief	Start the LE Audio role given by @p role.
  *
- * @note	Used by CIS.
- *
- * @param[in]	conn	The connection pointer.
- */
-void le_audio_conn_disconnected(struct bt_conn *conn);
-
-/**
- * @brief	Set pointer to extended advertisement with periodic adv configured
- *
- * @note	Used by BIS.
- *
- * @note	Will also start the broadcast_source.
- *
- * @param[in]	ext_adv	Pointer to the extended advertisement.
+ * @param[in]	Type of role to start, can only specify one at a time.
  *
  * @return	0 for success, error otherwise.
  */
-int le_audio_ext_adv_set(struct bt_le_ext_adv *ext_adv);
-
-/**
- * @brief	Get bt_data containing the data to advertise.
- *
- * @param[out]	adv		Pointer to the pointer of bt_data to advertise.
- * @param[out]	adv_size	Pointer to size of @p adv.
- * @param[in]	periodic	Specify if the data is for periodic advertisement.
- */
-void le_audio_adv_get(const struct bt_data **adv, size_t *adv_size, bool periodic);
-
-/**
- * @brief	Resume the Bluetooth LE Audio stream.
- *
- * @return	0 for success, error otherwise.
- */
-int le_audio_play(void);
-
-/**
- * @brief	Pause the Bluetooth LE Audio stream.
- *
- * @return	0 for success, error otherwise.
- */
-int le_audio_pause(void);
+int le_audio_start(enum audio_roles role);
 
 /**
  * @brief	Send the Bluetooth LE Audio data.
  *
  * @param[in]	enc_audio	Encoded audio struct.
+ * @param[in]	role		Specify what role to send through, can send to multiple roles.
  *
  * @retval	0	Operation successful
  * @retval	-ENXIO	The feature is disabled,
  * @retval	error	Otherwise.
```

Comment:
This looks strange

Reply:
Care to elaborate? Is it the formatting? The use of retval? I see that there are some periods missing